src/app/tools/qr-generator/components/QRCodeGenerator.tsx

```
@@ -1,10 +1,12 @@
 "use client";
 
-import React, { useState, useEffect, useRef, useCallback } from "react";
 import QRCode from "qrcode";
+import React, { useCallback, useEffect, useRef, useState } from "react";
```

Biome (2.4.15)

[error] These imports are unused.

(lint/correctness/noUnusedImports)